### PR TITLE
Fix React imports

### PR DESCRIPTION
**Pull request by upalatucci**
replace `import * as React` with `import React, {...} `

replace `React.FC` with `FC` and `React.useState` with `useState`

### CODE REVIEW

File: src/utils/components/PolicyForm/PolicyInterface.tsx [hunk 1]
```diff
@@ -51,10 +52,7 @@ const PolicyInterface: FC<PolicyInterfaceProps> = ({
   const [isTypeOpen, setTypeOpen] = useState(false);
   const [isAggregationOpen, setAggregationOpen] = useState(false);
 
-  const handleStateChange = (
-    event: React.MouseEvent<Element, MouseEvent>,
-    newState: NETWORK_STATES,
-  ) => {
+  const handleStateChange: SelectProps['onSelect'] = (event, newState: NETWORK_STATES) => {
```
Comment (avivtur) [inline src/utils/components/PolicyForm/PolicyInterface.tsx]: maybe replace `on-select` with const as it is used 3 times in here?

**Review by avivtur — COMMENTED** [review]
Great work Ugo! :100: 
Other than one really small comment everything looks great

File: src/utils/components/PolicyForm/PolicyInterface.tsx [hunk 2]
```diff
@@ -63,7 +61,7 @@ const PolicyInterface: FC<PolicyInterfaceProps> = ({
     onInterfaceChange((draftInterface) => (draftInterface.name = newName));
   };
 
-  const handleTypechange = (event: React.MouseEvent<Element, MouseEvent>, newType: string) => {
+  const handleTypechange: SelectProps['onSelect'] = (event, newType: string) => {
```
Comment (pcbailey) [inline src/utils/components/PolicyForm/PolicyInterface.tsx]: It looks like you removed the typing here, but left the import for `MouseEvent` causing the linter error above.

Comment (upalatucci) [reply src/utils/components/PolicyForm/PolicyInterface.tsx]: fixed

**Pull request merged**